### PR TITLE
Fixes for couple compatibility issues with latest images and other Rails fixes

### PR DESCRIPTION
**Pull request by mfojtik**
@ewolinetz @bparees PTAL

### CODE REVIEW

**Comment by jhadvig** [comment]
@mfojtik LGTM

File: example/template/ruby-rails-example.json [hunk 1]
```diff
@@ -1,171 +1,213 @@
...
       "generate": "expression",
-      "name": "SECRET_KEY_BASE"
+      "from": "[a-z0-9]{127}"
     }
   ],
   "labels": {
     "template": "application-template-stibuild"
   }
-}
+}
```
Comment (rhcarvalho) [inline example/template/ruby-rails-example.json]: @mfojtik ah! Your script to convert across versions is not writing the trailing newline :grinning:

Comment (mfojtik) [inline example/template/ruby-rails-example.json]: @rhcarvalho is that required in json?

**Comment by ewolinetz** [comment]
@mfojtik LGTM

File: example/template/ruby-rails-example.json (hunk 1 above)
Comment (rhcarvalho) [inline example/template/ruby-rails-example.json]: Just a UNIX convention, files end with "\n" (that's why GitHub, git and other tools will warn about it).
But I see you added one line too much, now it ends with an empty line "\n\n"...

**Pull request merged**